test/unit/client/test_client.py

```diff
@@ -61,6 +62,11 @@ def folder_id():
 
 
 @pytest.fixture(scope='module')
+def marker_id():
+    return 'marker_1'
+
+
+@pytest.fixture(scope='module')
```

Comment:
Won't this break if it is a module fixture? Because the `Mock` will retain its call spec between tests if we do this.